blueprints/typehints.py

```diff
@@ -5,3 +5,8 @@
 These constants are intended to improve code readability and prevent the use of 'magic numbers' in calculations.
 Distances are defined with postfixes indicating their units for clarity in usage throughout the code.
 """
+KN = float
+MM2 = float
+MPA = float
+
+KN_TO_N = 1e3
```

Comment:
do we want to place conversions (KN_TO_N) within the typehints.py or in unit_conversion.py?

Reply:
Or no unit conversions at all (stick to SI standard) : https://en.wikipedia.org/wiki/International_System_of_Units

Reply:
During the last meeting we saw that each eurocode has a unit definition at the top which is valid for the entire document. 
We thought it would be best to stick to these units and use these units as typehints, as explicit >> implicit. 

SI standard is really usefull for the back-end, but in a lot of cases this means a lot of conversion/difference with the units written down in the source documents. 

What do you think about this @bro-wi ? My personal preference is to make the unit explicit, eventhough the formula might be unitless, and stick to the unit used in the source document.

Reply:
I totally agree. My main concern is how we align formula's as input to a different formula in a intuitive way.

Reply:
The ouput unit of the formula is explicit as well so that would solve the problem.

Reply:
If we continue exploring the "back-trace intermediate calculations" we have to take care of a bit more I believe ;-)